src/Compilers/CSharp/Portable/Binder/Binder_Statements.cs

```diff
@@ -1593,6 +1593,21 @@ private static PropertySymbol GetPropertySymbol(BoundExpression expr, out BoundE
             return propertySymbol;
         }
 
+        internal static Symbol? GetIndexerSymbol(BoundExpression? e)
```

Comment:
>GetIndexerSymbol [](http://example.com/codeflow?start=32&length=16)

I think The name of the method is confusing. It is too general and doesn't reflect what can be returned at the same time.